media/stitcher/api/Stitcher.Samples.Tests/GetLiveAdTagDetailTests.cs

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Stitcher.Samples.Tests
+{
+    [Collection(nameof(StitcherFixture))]
+    public class GetLiveAdTagDetailTest : IAsyncLifetime
+    {
+        private StitcherFixture _fixture;
+        private readonly CreateLiveSessionSample _createSample;
+        private readonly GetLiveAdTagDetailSample _getSample;
+        private readonly ListLiveAdTagDetailsSample _listSample;
+
+        private string _liveSessionId;
+        private string _adTagDetailId;
+        private string _playUri;
+
+        public GetLiveAdTagDetailTest(StitcherFixture fixture)
+        {
+            _fixture = fixture;
+            _createSample = new CreateLiveSessionSample();
+            _getSample = new GetLiveAdTagDetailSample();
+            _listSample = new ListLiveAdTagDetailsSample();
+
+            var session = _createSample.CreateLiveSession(
+               _fixture.ProjectId, _fixture.LocationId, _fixture.LiveSourceUri, _fixture.LiveAdTagUri, _fixture.TestSlateId);
+            _liveSessionId = session.LiveSessionName.LiveSessionId;
+            _playUri = session.PlayUri;
+            Assert.NotNull(_playUri);
+        }
+        public async Task InitializeAsync()
+        {
+            // To get ad tag details, you need to curl the main manifest and
+            // a rendition first. This supplies media player information to the API.
+            //
+            // Curl the playUri first. The last line of the response will contain a
+            // renditions location. Curl the live session name with the rendition
+            // location appended.
+            string renditions = await _fixture.GetHttpResponse(_playUri);
+            Assert.NotNull(renditions);
+
+            Match m = Regex.Match(renditions, "renditions/.*", RegexOptions.IgnoreCase);
+            Assert.True(m.Success);
```

Comment:
In general abort the Asserts outside of the test itself. And we are not testing that renditions as where they should be, we are testing that we get the ad information, so we can assume the rest works. If the rest needs to be tested, then we should test that independently.

Reply:
Removed all Assert statements that are outside of tests.